qt/python/mantidqt/widgets/colorbar/colorbar.py

```diff
@@ -53,12 +55,12 @@ def __init__(self, parent=None, default_norm_scale=None):
         self.cmin_layout.addWidget(self.cmin)
         self.cmin_layout.addStretch()
 
+        self.linear_validator = QDoubleValidator(-sys.float_info.max, sys.float_info.max, 3, self)
```

Comment:
Shouldn't this just be `QDoubleValidator()` since -max -> max effectively means all representable double values